ovirtlib4/vms.py

```diff
@@ -97,6 +97,23 @@ def stop_and_wait(self, state='down', *args, **kwargs):
         vm = self.get(wait_for=f"entity.status.value == '{state}'", *args, **kwargs)
         return vm[0] if vm else None
 
+    def get_management_nic(self):
+        """
+        Gets the management vNIC of the VM
+
+        Args:
+            mng_network (str): Management network name at the Ovirt Engine
+
+        Returns:
+            VmNic: ovirtlib VmNic object if vNIC was found, None otherwise
+        """
+        vm_cluster = clusters.Clusters(connection=self.connection).get_entity_by_id(id=self.entity.cluster.id)
```

Comment:
It could have been great if you would wrap this up in a method and even add this as a property for the VM

Reply:
good idea!!! thx, I will associate this method with any object include a cluster property